### ISSUE TITLE
Issues with V1.7

### ISSUE THREAD

**Issue by cprgolds** [mracko/MSFS-Mobile-Companion-App#48]
Hi Martin,

I updated the Mobile Companion App to V1.7 today.  I am using MSFS 1.15.10.0

I have a slightly unusual setup where my Community folder is:  

_D:\AppData\Local\Packages\Flight_Sim_2020\Community_

The settings text file is the same as in your instructions, edited for username.

It seems to work okay on my computer if I use the url _localhost:4000_, however, I have not been able to get it to display on a laptop or tablet.  I just get a timing out trying to reach the server.

I reverted back to V1.5 and it displays fine on all of my devices.

Do you have any ideas of what is causing this?

Thanks!

RIck

**Comment by mracko** [comment]
Hi Rick,

This looks like an issue with Windows Defender. Make sure you white-list the 1.7 EXE in Windows Defender. I've already had a user with a similar issue - older version worked, the latest one didn't and the problem was with Windows Defender blocking the EXE.

The settings.txt file has nothing to do with connection issues. It's only relevant if you want to load the flight plan that you've created in the world map view into the app's moving map.

Please let me know if you managed to resolve the issue.

Thanks!
Martin

**Comment by cprgolds** [comment]
Hi Martin,

Problem is resolved.  Everything is working (including the settings file).

I am using Norton 360 instead of Windows Defender.  Yesterday, I had even completely disabled the firewall and still could not get MCA to work.  What I now suspect is that Windows Defender kicked in when windows saw no firewall.

Anyway I had to do two things in Norton..   First I had to allow the file to be trusted.  Norton was quarantining the exe when I tried to load it.  Secondly, as you suggested, I had to whitelist it to allow access access to communications.

THANKS!!

Rick
